frontend/cypress/integration/featureFiles/masthead.feature

```diff
@@ -0,0 +1,38 @@
+@masthead
+@multi-cluster
+# don't change first line of this file - the tag is used for the test scripts to identify the test suite
+
+Feature: Kiali masthead cluster components status
+
+  User wants to verify the Kiali masthead content
+
+  Background:
+    Given user is at administrator perspective
+    And user is at the "overview" page
+
+  @multi-cluster
+  Scenario: Cluster components healthy
+    Then user sees "east" cluster label with a "success" icon
+    When user hovers over the cluster icon
+    Then user sees a tooltip with text "east"
+    Then user sees a tooltip with text "west"
+    Then user does not see any "Not" in the tooltip
+    Then user does not see any "Unreachable" in the tooltip
+    Then user hovers out the cluster icon
+
+  @multi-cluster
+  @component-health
+  Scenario: Istio components unhealthy
+    When user scales to "0" the "prometheus" in namespace "istio-system"
```

Comment:
A faster way to do this is to just intercept and mock the status response to change one of them to unhealthy. I'm worried that actually scaling prom up/down could affect other tests after this one that rely on metrics data like graph tests.

Reply:
by mocking, we could not really test all those changes in the backend, it will be only UI functionality check,
but I agree that up/down of prometheus can cause some flake tests in backend, maybe I can up/down another component which is less sensitive, for instance 'grafana'

Reply:
@nrfox thanks for the review, made 'grafana' up/down scale instead of 'prometheus', and '@component-health-upscale' after operaton upscales it.